ci/utils.js

```diff
@@ -9,7 +9,7 @@ const url_path_join = function(...pieces) {
   // """
   const initial = pieces[0].startsWith("/");
   const final = pieces[pieces.length - 1].endsWith("/");
-  const result = pieces
+  let result = pieces
```

Comment:
I am a little confused — shouldn't this already be changed on master?

https://github.com/nteract/bookstore/blob/ad44c3abad19dce0a834e320cd30a5b18f2ccc29/ci/utils.js#L10-L15

Reply:
It is but this predated that so it is probably a merge conflict now that needs to be taken care of in this PR so that master is used.

Reply:
Rebased